swagger/Platfrom API/bizaiChats.json

```diff
@@ -0,0 +1,407 @@
+                    "total_pages": {
+                      "type": "integer"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "api_key": []
+          }
+        ],
+        "tags": [
+          "chatmng"
+        ],
+        "x-auth-type": "Directory"
+      },
+      "post": {
+        "operationId": "ChatsController_createChat",
+        "summary": "Create a chat",
+        "description": "Create a new chat session between a user and an agent",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "https://vcita.github.io/developers-hub/entities/bizaiChats.json"
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "201": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "data": {
+                      "$ref": "https://vcita.github.io/developers-hub/entities/bizaiChats.json"
+                    },
+                    "success": {
+                      "type": "boolean"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "api_key": []
+          }
+        ],
+        "tags": [
+          "chatmng"
+        ],
+        "x-auth-type": "Directory"
+      }
+    },
+    "/v3/bizai-chats/{uid}": {
+      "get": {
+        "operationId": "ChatsController_getChat",
+        "summary": "Retrieve a chat",
+        "description": "Get a specific chat session based on its unique identifier (UID)",
+        "parameters": [
+          {
+            "name": "uid",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The unique identifier of the chat"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Chat retrieved",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "success": {
+                      "type": "boolean"
+                    },
+                    "data": {
+                      "$ref": "https://vcita.github.io/developers-hub/entities/bizaiChats.json"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "api_key": []
+          }
+        ],
+        "tags": [
+          "chatmng"
+        ],
+        "x-auth-type": "Directory"
+      }
+    },
+    "/v3/bizai-chats/{uid}/messages": {
```

Comment:
do not nest the endpoint, it should be /v3/messages, you need to add the chat_uid to the path parameters